.github/workflows/astyle.yml

```diff
@@ -1,7 +1,12 @@
 name: astyle
 
 on:
-  pull_request
+  pull_request:
+    paths:
+    - '**.json'
```

Comment:
Probably not required

```suggestion
```

Reply:
Thank you for taking a look! The job in this file is called `style-json`, are you sure? Should the job be renamed?

Reply:
I suggest renaming the job; it seems to entirely be doing astyle, not checking JSON.

Reply:
done, thanks